scripts/vendor/select2.min.js

```diff
@@ -0,0 +1,2 @@
+/*! Select2 4.0.13 | https://github.com/select2/select2/blob/master/LICENSE.md */
```

Comment:
Do we really need yet another 3rd-party library added?

Reply:
No, not necessarily. What would be an alternative giving the same (or similar) function without an external library?

Reply:
I guess there isn't one. It's just the we keep adding more and more 3rd-party stuff, that's all :/

Reply:
Well, after all we're not doing this for the fun of it but because they provide new/better options. If the same thing can be achieved using stuff we already have onboard, I'd be all for it. However, if there isn't which seems to be the case, then importing a third-party library (which is independently tested and (hopefully) maintained) is surely better than reinventing the wheel ourselves.